packages/webapp/src/lib/services/RegistryManager.ts

```
@@ -0,0 +1,64 @@
+import { REGISTRY_URL } from '$lib/constants';
+
+export default class RegistryManager {
+	private static STORAGE_KEY = 'registry';
+
+	static getFromStorage(): string | null {
+		try {
+			return localStorage.getItem(this.STORAGE_KEY);
+		} catch (error) {
+			throw new Error('Failed to access localStorage: ' + (error instanceof Error ? error.message : String(error)));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)